.github/workflows/winget-release.yml

```diff
@@ -0,0 +1,29 @@
+name: Submit Winget Package to Windows Package Manager Community Repository
+on:
+  release:
+    types: [released]
+  workflow_dispatch:
+    inputs:
+      tag_name:
+        description: "Specific tag name"
+        required: false
+        type: string
+env:
+  CI: true
```

Comment:
why is that needed ?

Reply:
Indeed not needed, I was reusing another github worflow action.